db/migrate/20151124183140_remove_talks.rb

```diff
@@ -1,5 +1,6 @@
 class RemoveTalks < ActiveRecord::Migration
   def change
-    drop_table :talks
+    drop_table :talks do
```

Comment:
With the project I am working on right now, I had to do a drop_table. In the block, I specified the actual fields in the block. So the migration is reversible and what you see is what you get. Example: 

```
drop_table :users do |t|
  t.integer :id, index: true  
  t.string :username
  t.string :password
end
```

Of course, this is not the table I dropped.